### DISCUSSION TITLE
variantStudio

### DISCUSSION THREAD

**Discussion by dridk** [labsquare/CuteVariant-python#539] [General]
Just discover variantStudio from Illumina...

This is the most similar tools than cutevariant ! Lot of inspiration here !

https://support.illumina.com/content/dam/illumina-support/documents/documentation/software_documentation/variantstudio/variant-studio-v3-0-user-guide-1504890-01.pdf

**Comment by antonylebechec** [comment]
A interface idea should be to add a graph on Filters plugin, like:
<img width="1037" alt="image" src="https://user-images.githubusercontent.com/40463532/184556917-66c62eb0-651a-4a30-877c-3525aacbf844.png">

**Comment by antonylebechec** [comment]
Another is the "Annotation" plugin
